recipe/meta.yaml

```diff
@@ -1,34 +1,30 @@
+{% set name = "gcc_impl" %}
+{% set version = ctng_gcc %}
+{% set chost = ctng_cpu_arch ~ "-" ~ ctng_vendor ~ "-linux-gnu-" %}
 {% set build_num = ctng_gcc_build_number %}
 
 package:
-  name: ctng-compilers
-  version: {{ ctng_gcc }}
+  name: gcc_compilers
+  version: {{ version }}
 
 source:
-  path: .
+  - url: https://ftp.gnu.org/gnu/gcc/gcc-{{ version }}/gcc-{{ version }}.tar.gz
```

Comment:
We'll want the sha256 here before merging.

Reply:
Yes, will add that.